### DISCUSSION TITLE
private github repos - acess and usage?

### DISCUSSION THREAD

**Discussion by linoman-zz** [community/community#21752] [New to GitHub]
<p>hello dear experts</p>
<p>heard that we can run private github repos -</p>
<p>well; what about the acess and usage?</p>
<p>can i acess them in the same way like any github repo?</p>

**Comment by mpboom** [comment, marked as answer]
<p>Hi <a class="mention" href="/linoman-zz">@linoman-zz</a> ,</p>
<p>I’ll assume you are talking about private repositories on the  <strong>Free</strong> plan.</p>
<p>Yes, you can access them just like any other repository on GitHub. Usage is limited in the same way as all repositories on GitHub, free or paid. See <a href="https://help.github.com/en/articles/what-is-my-disk-quota" rel="nofollow noopener">https://help.github.com/en/articles/what-is-my-disk-quota</a> for more information about that. Free private repositories are limited in the amount of collaborators (max. 3) and some other features.</p>
<p>See <a href="https://github.com/pricing" rel="nofollow noopener">https://github.com/pricing</a> for more information on that matter.</p>
